rgw/v2/tests/s3_swift/test_Mbuckets_with_Nobjects.py

```diff
@@ -116,6 +117,38 @@ def test_exec(config):
                 raise TestExecError("RGW service restart failed")
             else:
                 log.info('RGW service restarted')
+        if config.gc_verification is True:
+            config.bluestore_block_size = 1549267441664
+            config.rgw_gc_max_queue_size = 367788
+            config.rgw_gc_processor_max_time = 3600
+            config.rgw_gc_max_concurrent_io = 10
+            config.rgw_gc_max_trim_chunk = 32
+            config.rgw_objexp_gc_interval = 10
+            config.rgw_gc_obj_min_wait = 10
+            config.rgw_gc_processor_period = 10
```

Comment:
Can these be taken from yaml config, if the values are present you can use these as default values

Reply:
Instead of taking it from yaml I would like to hard-code these values in script itself.

Reply:
That was not the intent of the my comment. Hardcodimg values in the code is always bad.
If the values are present in the config, then use them or use the values in the py file. A simple dictonary get() should help you to achieve this.

Reply:
gc values are now handled from yaml file

Reply:
script takes ceph conf values as a dictionary from yaml file now